draft-ietf-ppm-dap.md

```diff
@@ -1719,11 +1697,11 @@ where:
 * `public_share` is the report's public share
 * `plaintext_input_share` is the Leader's `PlaintextInputShare`
 
-The methods are defined in {{Section 5.8 of !VDAF}}. This process determines
-the initial per-report `state`, as well as the initial `outbound` message to
-send to the Helper. If `state` is of type `Rejected`, then the report is
+`ping_pong_leader_init` is defined in {{Section 5.7.1 of !VDAF}}. This process
+determines the initial per-report `state`, as well as the initial `outbound`
+message to the Helper. If `state` is of type `Rejected`, then the report is
 rejected and removed from the set of candidate reports, and no message is sent
-to the Helper.
+to the Helper for this report.
```

Comment:
clarity: we may still send an `AggregationJobInitReq` message, just not with this report in it